README.md

```diff
@@ -1,10 +1,66 @@
-# Math-Magicians
+# React and Redux
 
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
 
+## Set-up GitHub Actions
 
+This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.

```

Comment:
- [ ] Good job so far :+1: but your README file is not yet professional. Kindly use this [Microverse template](https://github.com/microverseinc/readme-template), edit it with your info and info about your project, and also add steps a user would need to set up and run your project locally :100: